Kruize remote monitoring functional tests failures due to performance_profile not found

**Describe the bug**
Kruize remote monitoring functional tests are failing with performance_profile not found from createExp and listRecommendations testcases on openshift with latest kruize 0.2 image 

https://ci.app-svc-perf.corp.redhat.com/job/ExternalTeams/job/Autotune/job/kruize_functional_tests/213/#showFailuresLink - Kruize scalelab

```
>       assert response.status_code == int(expected_status_code)
E       AssertionError: assert 400 == 201
E        +  where 400 = <Response [400]>.status_code
E        +  and   201 = int('201')

test_create_experiment.py:113: AssertionError
 ------------------------------Captured stdout call------------------------------ 

****************************************************
Test -  invalid_experiment_name
****************************************************

tmp_json_file =  /tmp/create_exp_invalid_experiment_name.json
IP =  kruize-openshift-tuning.apps.bm.example.com

KRUIZE AUTOTUNE URL =  http://kruize-openshift-tuning.apps.bm.example.com

Deleting the experiment...
URL =  http://kruize-openshift-tuning.apps.bm.example.com/createExperiment
<Response [400]>
Response status code =  400
delete exp =  400

************************************************************
[
    {
        "version": "v2.0",
        "experiment_name": "xyz",
        "cluster_name": "cluster-one-division-bell",
        "performance_profile": "resource-optimization-openshift",
        "mode": "monitor",
        "target_cluster": "remote",
        "kubernetes_objects": [
            {
                "type": "deployment",
                "name": "tfb-qrh-sample",
                "namespace": "default",
                "containers": [
                    {
                        "container_image_name": "kruize/tfb-qrh:1.13.2.F_et17",
                        "container_name": "tfb-server"
                    }
                ]
            }
        ],
        "trial_settings": {
            "measurement_duration": "15min"
        },
        "recommendation_settings": {
            "threshold": "0.1"
        }
    }
]

************************************************************

Creating the experiment...
URL =  http://kruize-openshift-tuning.apps.bm.example.com/createExperiment
Response status code =  400
{
    "message": "Not Found: performance_profile does not exist: resource-optimization-openshift",
    "httpcode": 400,
    "documentationLink": "",
    "status": "ERROR"
}
Not Found: performance_profile does not exist: resource-optimization-openshift
```

**How to reproduce it**
Run the remote monitoring functional tests with latest 0.2 build

**Expected behavior**
Performance profile should be created before running the tests and test cases should passed

**Relevant logs**
.

**Environment:**
- Kubernetes Cluster [e.g. minikube, openshift, kind]
- Version
- OS

**Additional context**
.

--- a/tests/scripts/common/common_functions.sh
+++ b/tests/scripts/common/common_functions.sh
@@ -1905,6 +1905,7 @@ function kruize_local_patch() {
 
 #
 # "local" flag is turned off for RM.
+# Restores kruize default cpu/memory resources, PV storage for openshift
 #
 function kruize_remote_patch() {
 	CRC_DIR="./manifests/crc/default-db-included-installation"
@@ -1916,6 +1917,8 @@ function kruize_remote_patch() {
     sed -i 's/"local": "true"/"local": "false"/' ${KRUIZE_CRC_DEPLOY_MANIFEST_MINIKUBE}
   elif [ ${cluster_type} == "openshift" ]; then
     sed -i 's/"local": "true"/"local": "false"/' ${KRUIZE_CRC_DEPLOY_MANIFEST_OPENSHIFT}
+    sed -i 's/\([[:space:]]*\)\(storage:\)[[:space:]]*[0-9]\+Mi/\1\2 1Gi/' ${KRUIZE_CRC_DEPLOY_MANIFEST_OPENSHIFT}
+    sed -i 's/\([[:space:]]*\)\(memory:\)[[:space:]]*".*"/\1\2 "2Gi"/; s/\([[:space:]]*\)\(cpu:\)[[:space:]]*".*"/\1\2 "2"/' ${KRUIZE_CRC_DEPLOY_MANIFEST_OPENSHIFT}
   fi
 }
 

